.gitmodules

```diff
@@ -809,3 +809,6 @@
 [submodule "vendor/grammars/reason"]
 	path = vendor/grammars/reason
 	url = https://github.com/facebook/reason
+[submodule "vendor/grammars/sublime-mzn"]
+	path = vendor/grammars/sublime-mzn
+	url = https://github.com/astenmark/sublime-mzn
```

Comment:
Did you compare this grammar with other concurrent MiniZinc grammars?
I found [this one for Atom](https://github.com/jjdekker/language-mzn). You can use [Lightshow](https://github-lightshow.herokuapp.com/) to compare them on sample files.